robot/Cumulus/tests/browser/levels/create_edit_delete_level.robot

```diff
@@ -8,115 +8,102 @@ Library         cumulusci.robotframework.PageObjects
+    [tags]                               W-038641                 feature:Levels
+
+
+    Go To Page                                          Home                      Level__c
     Enter Level Values
-    ...            Level Name=${level_name}
-    ...            Minimum Amount=0.1
-    ...            Maximum Amount=0.9
-    Enter Level Dd Values    Target    Contact
-    Enter Level Dd Values    Source Field    Total Gifts
-    Enter Level Dd Values    Level Field    Level
-    Enter Level Dd Values    Previous Level Field    Previous Level
-    Set Focus To Element   xpath: //input[@value='Save']
-    Click Button  Save
-    Current Page Should be    Details    Level__c
-    ${level_id} =   Save Current Record ID For Deletion  Level__c
-    Set Global Variable      ${level_name}
-    Set Global Variable      ${level_id}
-    Unselect Frame
-    Navigate To And Validate Field Value    Minimum Amount (>\=)   contains    0.10
-    Navigate To And Validate Field Value    Maximum Amount (<)    contains    0.90
-
-2 Edit Level and Verify Fields
-    # --------------------------------
-    # Modify the Level Values and validate that they save correctly
-    # --------------------------------
-    [tags]  unstable
-    Go To Page         Details         Level__c        object_id=${level_id}
-    Click Show More Actions Button    Edit
-    Wait Until Location Contains    /edit
-    Wait For Locator    frame    Levels
-    Choose Frame    Levels
-    Enter Level Dd Values    Source Field    Smallest Gift
+    ...                                                 Level Name=AutomationLevel
+    ...                                                 Minimum Amount=${min_amount}
+    ...                                                 Maximum Amount=${max_amount}
+    Enter Level Dd Values
+    ...                                                 Target=Contact
+    ...                                                 Source Field=Smallest Gift
+    ...                                                 Level Field=Level
+    ...                                                 Previous Level Field=Previous Level
+    Click Button                                        Save
+    Current Page Should be                              Details    Level__c
+    ${level_id} =                                       Save Current Record ID For Deletion  Level__c
+    Set Global Variable                                 ${level_id}
+    Go To Page                                          Details
+        ...                                             Level__c
+        ...                                             object_id=${level_id}
+
+    Wait Until Loading Is Complete
+    Navigate To And Validate Field Value                Minimum Amount (>\=)    contains    ${min_amount}
+    Navigate To And Validate Field Value                Maximum Amount (<)      contains    ${max_amount}
+    Go to edit level page                               ${level_id}
```

Comment:
highly recommend adding clicking on buttons to create New Level and edit level as they are loading different pages rather than just directly going to pages atleast once to verify that functionality as well

Reply:
Made the required changes to accommodate it